### DISCUSSION TITLE
Just trying to determine the use case for this

### DISCUSSION THREAD

**Discussion by allen-munsch** [abersheeran/a2wsgi#12] [General]
```
a2wsgi will not accumulate the requested content or response content in the memory
```
What is an example of how this could happen?

Was thinking maybe a large number of streaming requests? But I'm not sure. 

Also any plans on getting this merged back into uvicorn/fastapi, etc?

**Comment by abersheeran** [comment]
You can read [uvicorn/issue#371](https://github.com/encode/uvicorn/issues/371)

I have proposed PR, but they don’t seem to like it. So I don't want to initiate a PR again at this time. By the way, a2wsgi is not heavy, you can use it in any project.

**Reply by allen-munsch** [reply]
Thank you for the quick response. Yeah i was looking at it last night looks pretty compact for what's there.
